generators/contract_report.py

```
@@ -1,334 +1,447 @@
-1. Spring에서 계약서 데이터 수신 (ERD 기반)
-2. AI 모델로 법령 적법성 검토 수행  
-3. 검토 결과를 Spring 형태로 반환
-4. 이미지에서 보인 형태로 위반사항 포맷팅
+1. AI 생성 특약 + 계약서 기본정보 수신 및 파싱
+2. 계약서 적법성 검사 수행  
+3. 검사 결과를 Spring 형태로 반환
+4. 임대차보호법, 민법 위반사항 포맷팅
 """
 import sys
 import os
 from typing import List, Dict, Any, Optional
```

Ruff (0.12.2)

`typing.Optional` imported but unused

Remove unused import: `typing.Optional`

(F401)